podpac/core/coordinates/affine_coordinates.py

```diff
@@ -0,0 +1,349 @@
+from __future__ import division, unicode_literals, print_function, absolute_import
+
+from collections import OrderedDict
+
+import numpy as np
+import traitlets as tl
+import lazy_import
+
+rasterio = lazy_import.lazy_module("rasterio")
+affine_module = lazy_import.lazy_module("affine")
```

Comment:
Okay, I'll figure that out.

Reply:
The affine module is separate from rasterio, but rasterio has it as a dependency. Technically we can create and use affine coordinates without rasterio (I could replace `rasterio.Affine` with `affine.Affine` and remove rasterio from the coordinates). In that case we might want to add affine to dependencies.

Reply:
@mpu-creare I pushed a commit using affine.Affine instead of rasterio.Affine. I also added affine to the requirements. I think it's possible to move affine to extra requires somewhere, I'll let you decide on that.

Reply:
It's fine as a new requirement.